src/components/FirstTimeUserModal/FirstTimeUserModal.styles.jsx

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+
+export const FirstTimeUserModalWrapper = styled.section`
```

Comment:
I tested the modal on a smaller screensize view and it doesn't seem to be responsive. Could you try removing this wrapper and follow the pattern I'm using for the Statistics Modal [here](https://github.com/jamesurobertson/trumple/blob/master/src/components/Game.js#L66-L68)
The <Modal/> Component should work just by nesting the content of this modal inside.